xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd

```diff
@@ -405,12 +405,22 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="multimodal"/>
 			<xsd:enumeration value="busOnStreetBoarding">
 				<xsd:annotation>
-					<xsd:documentation>Bus stop directly on street. This means people waiting to board will literally be standing on the street.</xsd:documentation>
+					<xsd:documentation>Depreciated in favor of busStopWithinRoadwayBoarding.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="busStopWithinRoadwayBoarding">
+				<xsd:annotation>
+					<xsd:documentation>Bus stop directly on street. This means people waiting to board will literally be standing on the street. The usual elements of a stop (pole, bank, information) usually are present at the road side. Often there are marking on the floor and sometimes also traffic lights stopping the traffic for boarding and alighting.</xsd:documentation>
```

Comment:
Wouldn't busStopWithinRoadwayBoarding also have to be used to describe the situation where the boarding position (where to enter the vehicle) is on the street but the *waiting area* may be on a Quay at the opposite side of the street (i.e., passengers wait for the bus to arrive, then cross the street to board)? If yes, then the text should be adapted. 

The comment of @nick-knowles made me think about such cases and the level of detail one wants to describe Quays and Boarding Positions. I am not aware that separate modelling of waiting areas and boarding positions is foreseen. Such situations also exist (or at least used to exist) for small railway stations: one has to wait for the train to enter the station, and only then is allowed to go to the Quay where the train has stopped. Should this be modellable?

Reply:
Good point. And this is often the case. waiting is on the side of the street and a traffic light blocks the traffic for alighting/boarding. However, most models for classic public transport don't use BOARDING POSITION.